commands/bootstrap/bootstrap.go

```diff
@@ -76,6 +76,13 @@ func init() {
 	bootstrapCmd.PersistentFlags().StringVarP(&workingDirectoryPath,
 		"working_directory", "w", "work", "Working directory where to place deployment files")
 	viper.BindPFlag("working_directory", bootstrapCmd.PersistentFlags().Lookup("working_directory"))
+	bootstrapCmd.PersistentFlags().StringVarP(&deploymentName,
+		"deployment_name", "n", "", "Name of the deployment. If not specified deployment is based on time.")
```

Comment:
"If not specified deployment is based on time" => would replace by:
"If not specified, deployment name is based on time"